plugins/virtual_event/VirtualScheduleEvent/lib/model/filters/VirtualScheduleEventFilter.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @package plugins.virtualEvent
+ * @relatedService ScheduleEventService
+ * @subpackage model.filters.base
+ * @abstract
+ */
+class VirtualScheduleEventFilter extends ScheduleEventFilter
+{
+	public function init()
+	{
+		$this->fields = kArray::makeAssociativeDefaultValue(array(
+			'_eq_virtual_event_id',
+			'_in_virtual_event_id',
+			'_notin_virtual_event_id',
+			'_eq_virtual_schedule_event_sub_type'.
+			'_in_virtual_schedule_event_sub_type',
+			'_notin_virtual_schedule_event_sub_type',
+		), null);
+		
```

Comment:
make concatenation with the fields in the schedule event filter
array_merge like in mediaEntryFilterForPlaylist